src/cpp/DataBrokerParticipant.cpp

```diff
@@ -239,20 +239,14 @@ eprosima::fastrtps::rtps::GuidPrefix_t DataBrokerParticipant::guid()
     return eprosima::fastrtps::rtps::GUID_t().guidPrefix;
 }
 
-// TODO decide default QoS
 eprosima::fastdds::dds::DataWriterQos DataBrokerParticipant::default_datawriter_qos()
 {
-    eprosima::fastdds::dds::DataWriterQos dw;
-
-    dw.publish_mode().kind = eprosima::fastdds::dds::PublishModeQosPolicyKind::ASYNCHRONOUS_PUBLISH_MODE;
-
-    return dw;
+    return eprosima::fastdds::dds::DATAWRITER_QOS_DEFAULT;
```

Comment:
Every DataWriter, local or wan, must be asynchronous because of the design of this application. Thus this configuration should be added to the thos default method,

Reply:
Addressed in 0d71100